![Comparison between mouse, marmoset and human.](/assets/papers/spacetx.png)

### SpaceTx: A Roadmap for Benchmarking Spatial Transcriptomics Exploration of the Brain

**Authors:** B. Long, J. Miller, The SpaceTx Consortium  
**Journal:** arXiv - Quantitative Biology  
**Year:** 2023  
**DOI:** arXiv:2301.08436

**Abstract:**  
Mapping spatial distributions of transcriptomic cell types is essential to understanding the brain, with its exceptional cellular heterogeneity and the functional significance of its spatial organization. Spatial transcriptomics techniques are hoped to accomplish these measurements, but each method uses different experimental and computational protocols, with different trade- offs and optimizations. In 2017, the SpaceTx Consortium was formed to compare these methods and determine their suitability for large-scale spatial transcriptomic atlases. SpaceTx work included progress in tissue processing, taxonomy development, gene selection, image processing and data standardization, cell segmentation, cell type assignments, and visualization. Although the landscape of experimental methods has changed dramatically since the beginning of SpaceTx, the need for quantitative and detailed benchmarking of spatial transcriptomics methods in the brain is still unmet. Here, we summarize the work of SpaceTx and highlight outstanding challenges as spatial transcriptomics grows into a mature field. We also discuss how our progress provides a roadmap for benchmarking spatial transcriptomics methods in the future. Data and analyses from this consortium, along with code and methods are publicly available at https://spacetx.github.io/.

**Links:**

- [PDF](https://arxiv.org/pdf/2301.08436)
- [Website](https://spacetx.github.io/)
