<hr />
<div class="publications" id="publications">
	<div class="content-wrapper">
		<h1>Publications</h1>
		<div class="section">
			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/evoviewer.png" alt="EvoViewer overview screen." class="paper-img left" />
				</div>
				<h3>
					<span class="title">Cytosplore EvoViewer: Visual Analytics of Conserved Evolutionary Patterns in multi-species single-cell sequencing data</span>
					<span class="authors">S. Basu et al.</span>, IEEE Pacific Visualization Conference, 2025
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						Single-cell transcriptomics has enhanced our understanding of the brainâ€™s cellular composition. Biologists now analyze complex datasets to explore how marker genes influence biological processes, genetic variations, and phenotypic traits. A challenge is comparing these datasets across species to detect subtle differences or similarities to evolutionary development. Here, we present Cytosplore EvoViewer to facilitate examining relationships between transcriptomic datasets across species, simplifying the analysis of marker gene regulation and its impact on biological functions and integrating these findings with prior evolutionary knowledge or species-specific traits. We conducted a design study, including domain analysis, implementation of the results into Cytosplore EvoViewer, and an expert evaluation. Cytosplore EvoViewer offers valuable insights into genetic variations and evolutionary dynamics, helping to understand the diversity and the unity within diversity across species and their evolutionary development.
					</p>
				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://github.com/ManiVaultStudio/Cytosplore_EvoViewer/" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>
			<div class="paper">
				<div class="paper-image" id="science23">
					<img src="/assets/papers/simian.png" alt="Comparative transcriptomics reveals human-specific cortical features" class="paper-img right" />
				</div>
				<h3>
					<span class="title">Comparative transcriptomics reveals human-specific cortical features</span>
					<span class="authors">N.L. Jorstad et al.</span>, Science, 2023
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						The cognitive abilities of humans are distinctive among primates, but their molecular and cellular substrates are poorly understood. We used comparative single-nucleus transcriptomics to analyze samples of the middle temporal gyrus (MTG) from adult humans, chimpanzees, gorillas, rhesus macaques, and common marmosets to understand human-specific features of the neocortex. Human, chimpanzee, and gorilla MTG showed highly similar cell-type composition and laminar organization as well as a large shift in proportions of deep-layer intratelencephalic-projecting neurons compared with macaque and marmoset MTG. Microglia, astrocytes, and oligodendrocytes had more-divergent expression across species compared with neurons or oligodendrocyte precursor cells, and neuronal expression diverged more rapidly on the human lineage. Only a few hundred genes showed human-specific patterning, suggesting that relatively few cellular and molecular changes distinctively define adult human cortical structure.
					</p>
				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://www.science.org/doi/reader/10.1126/science.ade9516" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://www.science.org/doi/full/10.1126/science.ade9516" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
					<br style="clear:left;" />
					<br style="clear:left;" />
					<i>The Cytosplore Simian Viewer was developed as part of the article above. The viewer itself is described in more detail in <span class="title">Cytosplore Simian Viewer: Visual Exploration for Multi-Species Single-Cell RNA Sequencing Data</span> <span class="authors">S. Basu et al.</span>, VCBM 2023: Eurographics Workshop on Visual Computing for Biology and Medicine, 2023 <br/></i>
					<div class="paper-link">
						<a href="https://diglib.eg.org/bitstream/handle/10.2312/vcbm20231219/111-120.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://youtu.be/_RnTDDxs7oU?si=07u-5H4ErhAn7_B5" target="_blank">
							<i class="fas fa-file-video"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://graphics.tudelft.nl/Publications-new/2023/BEKJBLLH23/VCBMPresentationSimianViewer.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://diglib.eg.org/handle/10.2312/vcbm20231219" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://github.com/ManiVaultStudio/SimianViewerPlugin" target="_blank">
							<i class="fas fa-external-link"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>
			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/spacetx.png" alt="Comparison between mouse, marmoset and human." class="paper-img left" />
				</div>
				<h3>
					<span class="title">SpaceTx: A Roadmap for Benchmarking Spatial Transcriptomics Exploration of the Brain</span>
					<span class="authors">B. Long, J. Miller, The SpaceTx Consortium</span>, arXiv - Quantitative Biology, 2023
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						Mapping spatial distributions of transcriptomic cell types is essential to understanding the brain, with its exceptional cellular heterogeneity and the functional significance of its spatial organization. Spatial transcriptomics techniques are hoped to accomplish these measurements, but each method uses different experimental and computational protocols, with different trade-offs and optimizations. In 2017, the SpaceTx Consortium was formed to compare these methods and determine their suitability for large-scale spatial transcriptomic atlases. SpaceTx work included progress in tissue processing, taxonomy development, gene selection, image processing and data standardization, cell segmentation, cell type assignments, and visualization. Although the landscape of experimental methods has changed dramatically since the beginning of SpaceTx, the need for quantitative and detailed benchmarking of spatial transcriptomics methods in the brain is still unmet. Here, we summarize the work of SpaceTx and highlight outstanding challenges as spatial transcriptomics grows into a mature field. We also discuss how our progress provides a roadmap for benchmarking spatial transcriptomics methods in the future. <!--Data and analyses from this consortium, along with code and methods are publicly available at <a href="https://spacetx.github.io/">this https URL</a>.-->
					</p>
				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://arxiv.org/pdf/2301.08436" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://arxiv.org/archive/q-bio" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://spacetx.github.io/" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>
			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/hmm.png" alt="Comparison between mouse, marmoset and human." class="paper-img right" />
				</div>
				<h3>
					<span class="title">Comparative cellular analysis of motor cortex in human, marmoset, and mouse</span>
					<span class="authors">Bakken et al.</span>, Nature, 2021.
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						The primary motor cortex (M1) is essential for voluntary fine motor control and is functionally conserved across mammals. Using high-throughput transcriptomic and epigenomic profiling of over 450,000 single nuclei in human, marmoset monkey, and mouse, we demonstrate a broadly conserved cellular makeup of this region, whose similarity mirrors evolutionary distance and is consistent between the transcriptome and epigenome. The core conserved molecular identity of neuronal and non-neuronal types allowed the generation of a cross species consensus cell type classification and inference of conserved cell type properties across species. Despite overall conservation, many species specializations were apparent, including differences in cell type proportions, gene expression, DNA methylation, and chromatin state. Few cell type marker genes were conserved across species, providing a short list of candidate genes and regulatory mechanisms responsible for conserved features of homologous cell types, such as the GABAergic chandelier cells. This consensus transcriptomic classification allowed the Patch-seq identification of layer 5 (L5) corticospinal Betz cells in non human primate and human and characterization of their highly specialized physiology and anatomy. These findings highlight the robust molecular underpinnings of cell type diversity in M1 across mammals and point to the genes and regulatory pathways responsible for the functional identity of cell types and their species specific adaptations.
					</p>
				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://www.nature.com/articles/s41586-021-03465-8.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://www.nature.com/articles/s41586-021-03465-8" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>
			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/mtg.jpg" alt="Cytosplore Viewer" class="paper-img left" />
				</div>
				<h3>
					<span class="title">Conserved Cell Types With Divergent Features in Human Versus Mouse Cortex.</span>
					<span class="authors">Hodge, Bakken et al.</span>, Nature, 2019.
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						Elucidating the cellular architecture of the human cerebral cortex is central to understanding our cognitive abilities and susceptibility to disease. Here we used single-nucleus RNA-sequencing analysis to perform a comprehensive study of cell types in the middle temporal gyrus of human cortex. We identified a highly diverse set of excitatory and inhibitory neuron types that are mostly sparse, with excitatory types being less layer-restricted than expected. Comparison to similar mouse cortex single-cell RNA-sequencing datasets revealed a surprisingly well-conserved cellular architecture that enables matching of homologous types and predictions of properties of human cell types. Despite this general conservation, we also found extensive differences between homologous human and mouse cell types, including marked alterations in proportions, laminar distributions, gene expression and morphology. These species-specific features emphasize the importance of directly studying human brain.
					</p>
				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://www.biorxiv.org/content/early/2018/08/05/384826.full.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://vimeo.com/273182940" target="_blank">
							<i class="fas fa-file-video"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://doi.org/10.1101/384826" target="_blank">
							<i class="fas fa-file-video"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://vimeo.com/273183030" target="_blank">
							<i class="fas fa-file-video"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://doi.org/10.1038/s41586-019-1506-7" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>
			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/mouse.jpg" alt="Cytosplore Viewer" class="paper-img right" />
				</div>
				<h3>
					<span class="title">Shared and Distinct Transcriptomic Cell Types Across Neocortical Areas.</span>
					<span class="authors">Tasic et al.</span>, Nature, 2018.
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						The neocortex contains a multitude of cell types that are segregated into layers and functionally distinct areas. To investigate the diversity of cell types across the mouse neocortex, here we analysed 23,822 cells from two areas at distant poles of the mouse neocortex: the primary visual cortex and the anterior lateral motor cortex. We define 133 transcriptomic cell types by deep, single-cell RNA sequencing.

					</p>
				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://www.nature.com/articles/s41586-018-0654-5.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://doi.org/10.1038/s41586-018-0654-5" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>
		</div>
	</div>
</div>