<hr />
<div class="publications" id="publications">
	<div class="content-wrapper">
		<h1>Publications</h1>
		<div class="section">

			<div class="paper">
				<div class="paper-image" id="science23">
					<img src="/assets/papers/simian.png"
						alt="Comparative transcriptomics reveals human-specific cortical features"
						class="paper-img left" />
				</div>
				<h3>
					<span class="title">Comparative transcriptomics reveals human-specific cortical features</span>
					<span class="authors">N.L. Jorstad et al.</span>, BioRxiv, 2022
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						Humans have unique cognitive abilities among primates, including language, but their molecular,
						cellular, and circuit substrates are poorly understood. We used comparative single nucleus
						transcriptomics in adult humans, chimpanzees, gorillas, rhesus macaques, and common marmosets
						from the middle temporal gyrus (MTG) to understand human-specific features of cellular and
						molecular organization. Human, chimpanzee, and gorilla MTG showed highly similar cell type
						composition and laminar organization, and a large shift in proportions of deep layer
						intratelencephalic-projecting neurons compared to macaque and marmoset. Species differences in
						gene expression generally mirrored evolutionary distance and were seen in all cell types,
						although chimpanzees were more similar to gorillas than humans, consistent with faster
						divergence along the human lineage. Microglia, astrocytes, and oligodendrocytes showed
						accelerated gene expression changes compared to neurons or oligodendrocyte precursor cells,
						indicating either relaxed evolutionary constraints or positive selection in these cell types.
						Only a few hundred genes showed human-specific patterning in all or specific cell types, and
						were significantly enriched near human accelerated regions (HARs) and conserved deletions
						(hCONDELS) and in cell adhesion and intercellular signaling pathways. These results suggest that
						relatively few cellular and molecular changes uniquely define adult human cortical structure,
						particularly by affecting circuit connectivity and glial cell function.

						<a href="https://www.biorxiv.org/content/10.1101/2022.09.19.508480v1" target="_blank">...</a>
					</p>

				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://www.biorxiv.org/content/10.1101/2022.09.19.508480v1.full.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://www.biorxiv.org/content/10.1101/2022.09.19.508480v1" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://github.com/ManiVaultStudio/SimianViewerPlugin" target="_blank">
							<i class="fas fa-external-link"></i>
						</a>
					</div>

					<a href="https://sec.lumc.nl/mtg-viewer/viewer/win/SV/install_cytosplore_simian_viewer_offline.exe"
						target="_blank">
						<i class="fas fa-download"> Download Cytosplore Simian Viewer for Windows</i>
					</a>

				</div>

				<br style="clear:left;" />

			</div>

			<br />

			<div class="paper">
				<div class="paper-image" id="simian-pub">
					<img src="/assets/papers/simian.png"
						alt="Cytosplore Simian Viewer: Visual Exploration for Multi-Species Single-Cell RNA Sequencing Data."
						class="paper-img right" />
				</div>
				<h3>
					<span class="title">Cytosplore Simian Viewer: Visual Exploration for Multi-Species Single-Cell RNA
						Sequencing Data</span>
					<span class="authors">S. Basu et al.</span>, VCBM 2023: Eurographics Workshop on Visual Computing
					for Biology and Medicine , 2023
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						With the rapid advances in single-cell sequencing technologies, novel types of studies into the
						cell-type makeup of the brain have become possible. Biologists often analyze large and complex
						single-cell transcriptomic datasets to enhance knowledge of the intricate features of cellular
						and molecular tissue organization. A particular area of interest is the study of whether cell
						types and their gene regulation are conserved across species during evolution. However, in-depth
						comparisons across species of such high-dimensional, multi-modal single-cell data pose
						considerable visualization challenges. This paper introduces Cytosplore Simian Viewer, a
						visualization system that combines various views and linked interaction methods for comparative
						analysis of single-cell transcriptomic datasets across multiple species. Cytosplore Simian
						Viewer enables biologists to help gain insights into the cell type and gene expression
						differences and similarities among different species, particularly focusing on comparing human
						data to other species. The system validation in discovery research on real-world datasets
						demonstrates its utility in visualizing valuable results related to the evolutionary development
						of the middle temporal gyrus

						<a href="https://diglib.eg.org/handle/10.2312/vcbm20231219" target="_blank">...</a>
					</p>

				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://diglib.eg.org/bitstream/handle/10.2312/vcbm20231219/111-120.pdf"
							target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://youtu.be/_RnTDDxs7oU?si=07u-5H4ErhAn7_B5" target="_blank">
							<i class="fas fa-file-video"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://graphics.tudelft.nl/Publications-new/2023/BEKJBLLH23/VCBMPresentationSimianViewer.pdf"
							target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://diglib.eg.org/handle/10.2312/vcbm20231219" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://github.com/ManiVaultStudio/SimianViewerPlugin" target="_blank">
							<i class="fas fa-external-link"></i>
						</a>
					</div>

					<a href="https://sec.lumc.nl/mtg-viewer/viewer/win/SV/install_cytosplore_simian_viewer_offline.exe"
						target="_blank">
						<i class="fas fa-download"> Download Cytosplore Simian Viewer for Windows</i>
					</a>

				</div>

				<br style="clear:left;" />

			</div>

			<br />

			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/spacetx.png" alt="Comparison between mouse, marmoset and human."
						class="paper-img left" />
				</div>
				<h3>
					<span class="title">SpaceTx: A Roadmap for Benchmarking Spatial Transcriptomics Exploration of the
						Brain</span>
					<span class="authors">B. Long, J. Miller, The SpaceTx Consortium</span>, arXiv - Quantitative
					Biology, 2023
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						Mapping spatial distributions of transcriptomic cell types is essential to understanding the
						brain, with its exceptional cellular heterogeneity and the functional significance of its
						spatial organization. Spatial transcriptomics techniques are hoped to accomplish these
						measurements, but each method uses different experimental and computational protocols, with
						different trade-offs and optimizations. In 2017, the SpaceTx Consortium was formed to compare
						these methods and determine their suitability for large-scale spatial transcriptomic atlases.
						SpaceTx work included progress in tissue processing, taxonomy development, gene selection, image
						processing and data standardization, cell segmentation, cell type assignments, and
						visualization. Although the landscape of experimental methods has changed dramatically since the
						beginning of SpaceTx, the need for quantitative and detailed benchmarking of spatial
						transcriptomics methods in the brain is still unmet. Here, we summarize the work of SpaceTx and
						highlight outstanding challenges as spatial transcriptomics grows into a mature field. We also
						discuss how our progress provides a roadmap for benchmarking spatial transcriptomics methods in
						the future. Data and analyses from this consortium, along with code and methods are publicly
						available at <a href="https://spacetx.github.io/">this https URL</a>.

						<a href="https://arxiv.org/archive/q-bio" target="_blank">...</a>
					</p>

				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://arxiv.org/pdf/2301.08436" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://diglib.eg.org/handle/10.2312/vcbm20231219" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>

			<br />

			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/hmm.png" alt="Comparison between mouse, marmoset and human."
						class="paper-img right" />
				</div>
				<h3>
					<span class="title">Comparative cellular analysis of motor cortex in human, marmoset, and
						mouse</span>
					<span class="authors">Bakken et al.</span>, Nature, 2021.
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						The primary motor cortex (M1) is essential for voluntary fine motor control and is functionally
						conserved across mammals. Using high-throughput transcriptomic and epigenomic profiling of over
						450,000 single nuclei in human, marmoset monkey, and mouse, we demonstrate a broadly conserved
						cellular makeup of this region, whose similarity mirrors evolutionary distance and is consistent
						between the transcriptome and epigenome. The core conserved molecular identity of neuronal and
						non-neuronal types allowed the generation of a cross species consensus cell type classification
						and inference of conserved cell type properties across species. Despite overall conservation,
						many species specializations were apparent, including differences in cell type proportions, gene
						expression, DNA methylation, and chromatin state. Few cell type marker genes were conserved
						across species, providing a short list of candidate genes and regulatory mechanisms responsible
						for conserved features of homologous cell types, such as the GABAergic chandelier cells. This
						consensus transcriptomic classification allowed the Patch-seq identification of layer 5 (L5)
						corticospinal Betz cells in non human primate and human and characterization of their highly
						specialized physiology and anatomy. These findings highlight the robust molecular underpinnings
						of cell type diversity in M1 across mammals and point to the genes and regulatory pathways
						responsible for the functional identity of cell types and their species specific adaptations.
						<a href="https://www.nature.com/articles/s41586-021-03465-8" target="_blank">...</a>
					</p>
				</div>

				<div class="paper-links">
					<div class="paper-link">
						<a href="https://www.nature.com/articles/s41586-021-03465-8.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://www.nature.com/articles/s41586-021-03465-8" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>
			<br />

			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/mtg.jpg" alt="Cytosplore Viewer" class="paper-img left" />
				</div>
				<h3>
					<span class="title">Conserved Cell Types With Divergent Features in Human Versus Mouse
						Cortex.</span>
					<span class="authors">Hodge, Bakken et al.</span>, Nature, 2019.
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						Elucidating the cellular architecture of the human cerebral cortex is central to understanding
						our cognitive abilities and susceptibility to disease. Here we used single-nucleus
						RNA-sequencing analysis to perform a comprehensive study of cell types in the middle temporal
						gyrus of human cortex. We identified a highly diverse set of excitatory and inhibitory neuron
						types that are mostly sparse, with excitatory types being less layer-restricted than expected.
						Comparison to similar mouse cortex single-cell RNA-sequencing datasets revealed a surprisingly
						well-conserved cellular architecture that enables matching of homologous types and predictions
						of properties of human cell types. Despite this general conservation, we also found extensive
						differences between homologous human and mouse cell types, including marked alterations in
						proportions, laminar distributions, gene expression and morphology. These species-specific
						features emphasize the importance of directly studying human brain
						<a href="https://doi.org/10.1101/384826" target="_blank"> ... </a>
					</p>
				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://www.biorxiv.org/content/early/2018/08/05/384826.full.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://vimeo.com/273182940" target="_blank">
							<i class="fas fa-file-video"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://vimeo.com/273183030" target="_blank">
							<i class="fas fa-file-video"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://doi.org/10.1038/s41586-019-1506-7" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>

			<br />

			<div class="paper">
				<div class="paper-image">
					<img src="/assets/papers/mouse.jpg" alt="Cytosplore Viewer" class="paper-img right" />
				</div>
				<h3>
					<span class="title">Shared and Distinct Transcriptomic Cell Types Across Neocortical Areas.</span>
					<span class="authors">Tasic et al.</span>, Nature, 2018.
				</h3>
				<div class="section abstract">
					<p>
						<b>Abstract:</b>
						The neocortex contains a multitude of cell types that are segregated into layers and
						functionally distinct areas. To investigate the diversity of cell types across the mouse
						neocortex, here we analysed 23,822 cells from two areas at distant poles of the mouse neocortex:
						the primary visual cortex and the anterior lateral motor cortex. We define 133 transcriptomic
						cell types by deep, single-cell RNA sequencing
						<a href="https://doi.org/10.1038/s41586-018-0654-5" target="_blank">...</a>
					</p>
				</div>
				<div class="paper-links">
					<div class="paper-link">
						<a href="https://www.nature.com/articles/s41586-018-0654-5.pdf" target="_blank">
							<i class="fas fa-file-pdf"></i>
						</a>
					</div>
					<div class="paper-link">
						<a href="https://doi.org/10.1038/s41586-018-0654-5" target="_blank">
							<i class="fas fa-external-link-alt"></i>
						</a>
					</div>
				</div>
				<br style="clear:left;" />
			</div>

		</div>
	</div>
</div>